docs/en/contributor/Building-Alluxio-From-Source.md

```diff
@@ -67,15 +77,15 @@ $ ./bin/alluxio format
 $ ./bin/alluxio-start.sh local SudoMount
 ```
 
-To verify that Alluxio is running, you can visit [http://localhost:19999](http://localhost:19999) or check the log in the `alluxio/logs` directory. You can also run a simple program:
+To verify that Alluxio is running, you can visit [http://localhost:19999](http://localhost:19999) or check the log in the `alluxio/logs` directory. It may take a few seconds for the web server to start. You can also run a simple program to test your environment:
```

Comment:
`runTests` tests that data can be written to and read from alluxio and the UFS. There is a different `validateEnv` command for validating the environment.